solutions/hpc/catalogValidationValues.json.template

```diff
@@ -0,0 +1,10 @@
+{
+  "ibmcloud_api_key": $VALIDATION_APIKEY,
+  "prefix": $PREFIX,
+  "zones": $ZONES,
+  "resource_group": $RG_NAME,
+  "bastion_ssh_keys": $SSH_KEYS,
+  "login_ssh_keys": $SSH_KEYS,
+  "compute_ssh_keys": $SSH_KEYS,
+  "storage_ssh_keys": $SSH_KEYS
```

Comment:
`$SSH_KEYS` is also not supported. In SLZ we us the pre-validation feature to generate a unique SSH key before validation begins. Here is the script we use -> https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/blob/main/tests/scripts/pre-validation-generate-ssh-key.sh

It basically generates an SSH key pair, and appends it to the `catalogValidationValues.json` that gets generated by this template file in the pipeline. For more info, see https://github.ibm.com/GoldenEye/documentation/blob/master/catalog.md#5-optional-add-pre-validation-or-post-validation

Reply:
To be fixed. Temp fix: hardcoded the values.